xml/vt_docker_quick.xml

```diff
@@ -823,9 +823,51 @@ RUN zypper -n in vim</screen>
   </para>
 
   <para>
+    Portus is accessible for &slea;&nbsp;12 customers through the Containers
+    module. This means that you should perform the following command to install
+    Portus:
+  </para>
+
+  <screen>sudo zypper in Portus</screen>
+
+  <para>
+    After doing this, you should proceed on configuring your freshly installed
+    Portus instance. This is thoroughly documented here:
+    <link xlink:href="http://port.us.org/docs/setups/1_rpm_packages.html#portus-dependencies"/>.
+    This document will help you to get through the installation process, and
+    it will also warn you about some of the common pitfalls.
+  </para>
+
+  <para>
+    After installing Portus, you might want to configure it. Note that the
+    portusctl command has already done most of the needed configuration.
+    However, there are some configurable values that you should have to take a
+    look before using Portus. This is thoroughly explained in this
+    documentation page:
+    <link xlink:href="http://port.us.org/docs/Configuring-Portus.html"/>.
+  </para>
+
+  <para>
+    After doing this, you should restart Apache so Portus fetches what you have
+    changed in the configuration. Note that you will always have to restart
+    Apache after changing a configuration value.
+  </para>
+
+  <para>
+    Finally, when entering Portus for the first time, you will be required
+    to enter some information about your installed registry. We have written
+    down the process about this in the following section of our documentation:
+    <link xlink:href="http://port.us.org/docs/setups/1_rpm_packages#the-default-installation.html"/>.
+  </para>
+
+  <para>
+    At this point, you should be able to enjoy Portus!
+  </para>
```

Comment:
Maybe rephrase to: The Portus setup is now complete and you can start using Portus.